src/NuGet.Core/NuGet.Commands/NuGet.CommandLine/SourcesListFormat.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.CommandLine
```

Comment:
You can change the namespace to something more proper.

Reply:
NuGet.Commands